src/main/java/training/Robot.java

```diff
@@ -1,11 +1,15 @@
 package training;
 
+import training.subsystems.elbow.Elbow;
+
 public class Robot {
 
 	// Enter your subsystems...
+	private Elbow elbow;
 
 	public Robot() {
 		// Boot your subsystems...
+		elbow = Elbow.getInstance();
```

Comment:
consider just doing `Elbow.init();` because it also intializes the elbow but it doesn't return the instance.